spec/requests/charges_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe "Charges pages" do
```

Comment:
There is should be presented *main* describe block that has nested describe blocks.
When you create new factory object in the main describe, it's also available in the his child blocks. Same with `before { }` .